tests/integration/states/test_resource_group.py

```diff
@@ -73,7 +75,8 @@ def test_changes(salt_call_cli, resource_group, location, tags, connection_auth)
     assert data == expected
 
 
-@pytest.mark.run(order=-1)
+# @pytest.mark.run(order=-1)
+@pytest.mark.skip(reason="do not delete this rg during testing")
```

Comment:
**chore (blocking):** if we're skipping a test in _all_ scenarios, we should remove it.